openstack_plugin_common/__init__.py

```diff
@@ -31,6 +32,8 @@
 from cloudify import context
 from cloudify.exceptions import NonRecoverableError, RecoverableError
 
+warnings.simplefilter('once')
```

Comment:
Does it only filter the same warning if it appeared more than once or is it general for all filters?

Couldn't we use the filter specifically where the relevant warning appears using https://docs.python.org/2/library/warnings.html#temporarily-suppressing-warnings?